apps/core/components/passengerForm/PassengerForm.js

```diff
@@ -191,7 +212,11 @@ class PassengerForm extends React.Component<Props, State> {
           </View>
           <View style={styles.spacer} />
           <View style={styles.menuButtonWrap}>
-            <Button block={true} label="Save" onPress={this.handleSavePress} />
+            <Button
+              block={true}
+              label={this.props.isEditing ? 'Edit' : 'Save'}
```

Comment:
Not sure about this. Original `Save` is ok for me in both cases. When I see `Edit` button I would expect to be taken to some other "edit screen" not that it will save my updated values.
But maybe I just get used too much to that old `Save` label. 😄

Reply:
What about `Save Edit` instead of `Edit`? 😆

Reply:
Not sure 🙂, I'm for good old `Save` personally. But I'm also definityle open to other opinions. Lets ask at least one more person 😉

Reply:
Let's keep `Save` then, reverted.